src/openforms/prefill/contrib/customer_interactions/api/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+
+from .views import CommunicationPreferencesView
+
+app_name = "prefill_customer_interactions"
+
+
+urlpatterns = [
+    path(
+        "communication-preferences/<uuid:submission_uuid>/component/<slug:profile_component>",
```

Comment:
can you add a test case with a component key `foo.bar`? I'm not sure if the slug pattern accepts that, I'd expect it doesn't.